profiles/apparmor.d/usr.share.openqa.script.openqa

```diff
@@ -19,8 +19,7 @@
   /etc/openqa/database.ini r,
   /etc/openqa/openqa.ini r,
   /proc/meminfo r,
-  /tmp/mojo.tmp rw,
-  /tmp/mojo.tmp.* rw,
+  /tmp/** rw,
   /usr/bin/optipng rix,
```

Comment:
if this is not what we want, I can also give the mktemp a template so we can limit it to /tmp/openqa.* - would that make sense?